data/campaigns/tutorial/utils/utils.cfg

```diff
@@ -5,7 +5,8 @@
         text={STRING}
         size=18
         duration=10000
```

Comment:
Whether or not this PR goes in, I feel that the tutorial should probably use the default values. The default duration is currently 50ms, which I consider to be a bug (introduced in a9d9e48c72e70fd41ae9759e3894752eb9708596).

Reply:
I'm not sure if it makes sense for the tutorial to use the default values. I guess if it depends if your expected common use case is the same as how the tutorial uses it?

Reply:
50ms doesn't make sense at all.

Reply:
Resolved in #5868.